cpanfile

```diff
@@ -4,6 +4,7 @@ requires 'XML::LibXML';
 requires 'XML::Writer';
 requires 'XML::Simple';
 requires 'IO::File';
+requires 'IO::Socket::PortState';
```

Comment:
remember that workers don't care about cpanfile. If you merge this you will break basically all tests on o3 *and* osd

Reply:
Hence the salt recipe change. We can think about running isotovideo in a container where we install with cpan :)